rules/library.bzl

```diff
@@ -524,12 +524,25 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
             namespace = namespace,
             hdrs = [],
             direct_hdr_providers = [swift_libname],
-            flatten_headers = True,
+            flatten_headers = False,
             tags = _MANUAL,
         )
         internal_deps.append(swift_doublequote_hmap_name)
         _append_headermap_copts(swift_doublequote_hmap_name, "-iquote", objc_copts, swift_copts, cc_copts)
 
+        # Add generated swift header to header maps for double quote imports
+        swift_angle_bracket_hmap_name = name + "_swift_angle_bracket_hmap"
```

Comment:
so this is for imports with format `<some imports>`? Also wonder why the comment suggests `double quote` imports

Reply:
bad copy/paste, it should say angle brackets. ill fix